compiler/src/org.graalvm.compiler.hotspot/src/org/graalvm/compiler/hotspot/meta/HotSpotGraphBuilderPlugins.java

```diff
@@ -563,6 +566,37 @@ private static void registerCounterModePlugins(InvocationPlugins plugins, GraalH
         }
     }
 
+    private static void registerBase64Plugins(InvocationPlugins plugins, GraalHotSpotVMConfig config, MetaAccessProvider metaAccess, ForeignCallsProvider foreignCalls) {
+        if (config.useBase64Intrinsics()) {
+            Registration r = new Registration(plugins, "java.util.Base64.Encoder");
```

Comment:
`java.util.Base64$Encoder` should be fine. It is using `Class.forName` for resolving the class.